src/System Application/Test/Email/src/EmailTest.Codeunit.al

```diff
@@ -300,7 +300,6 @@ codeunit 134685 "Email Test"
         Assert.IsFalse(EmailEditor.BccField.Editable(), 'Bcc field was editable');
         Assert.IsFalse(EmailEditor.SubjectField.Editable(), 'Subject field was editable');
         Assert.IsFalse(EmailEditor.BodyField.Editable(), 'Body field was editable');
-        Assert.IsFalse(EmailEditor.Attachments.Upload.Visible(), 'Upload Action is visible.');
```

Comment:
wrap them with `#if not CLEAN25` instead, the functionality will still exist for a while, we can keep the tests for the same period of time